### PR TITLE
Use natario1/CameraView instead of unmaintained google/CameraView

### PR DESCRIPTION
**Pull request by lytefast**
The original [google/CameraView](https://github.com/google/cameraview)  project we were using has not had any active development or recent commits in a while. Meanwhile a fork from said project has been updated and is more actively developed here: https://github.com/natario1/CameraView/

This commit switches to the active project in hopes that:
- PRs can be upstreamed quicker without issues and remove the need for a patched aar binary
- bugs from other developers are pulled in easily
- API contract is cleaner

This should address the issue #37 (see here for alternatives considered)

### CODE REVIEW

File: cameraview-aar/build.gradle [hunk 1]
```diff
@@ -1,2 +0,0 @@
-configurations.maybeCreate("default")
-artifacts.add("default", file('cameraview-patched.aar'))
```
Comment (mrkcsc) [inline cameraview-aar/build.gradle:2]: woot

File: flexinput/build.gradle [hunk 2]
```diff
@@ -19,10 +19,12 @@ android {
 
 dependencies {
     compile fileTree(include: ['*.jar'], dir: 'libs')
```
Comment (mrkcsc) [inline flexinput/build.gradle]: this line isn't needed iirc since we don't bundle and jars or libs.

**Review by mrkcsc — APPROVED** [review]
Swap out looks clean, excited to get this out in the wild.

File: flexinput/build.gradle (hunk 2 above)
Comment (lytefast) [reply flexinput/build.gradle]: yep your right

**Comment by lytefast** [comment]
on further review it seems like this project doesn't support Camera2 API either. This is kind of a deal killer for me as it provides a better API that's less error prone, and also perf benefits

**Comment by lytefast** [comment]
closing this out. Chose to use CameraKit due to the amount of update/support it has.

**Pull request closed without merging**